storagetransfer/storagetransfer_test.go

```diff
@@ -188,6 +189,120 @@ func TestGetLatestTransferOperation(t *testing.T) {
 	}
 }
 
+func TestDownloadToPosix(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	buf := new(bytes.Buffer)
+
+	rootDirectory, err := ioutil.TempDir("", "download-to-posix-test")
+	if err != nil {
+		t.Errorf("download_to_posix: %#v", err)
```

Comment:
This should probably be t.Fatalf since the test can't proceed if this fails. Same for L210 and elsewhere in other samples